CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
   updates to existing configurations.
 - `PATCH /network-interfaces/{id}` only allowed post-boot. Use `PUT` for
   pre-boot updates to existing configurations.
+- Within guest, MMDS response content has JSON format.
```

Comment:
The changelog entry should be added in the commit that adds the code. The are a few reasons for this. If we want to ever revert this change, we have to go through multiple commits. It keeps the history tidy and easier to read as unit changes are contained in a single commit.